DQMOffline/Trigger/python/TopMonitor_cfi.py

```diff
@@ -36,6 +36,12 @@
   xmax  = cms.double( 1.0  ),
 )
 
+#Suvankar
+hltTOPmonitoring.applyleptonPVcuts = cms.bool(False)
+hltTOPmonitoring.leptonPVcuts = cms.PSet(
+  dxy = cms.double(   0   ),
+  dz  = cms.double(   0   ),
```

Comment:
@sroychow same comment as above: please set the two parameter to some safe value